docs/source/fdtd101.rst

```diff
@@ -0,0 +1,29 @@
+********
+FDTD 101
+********
+
+FDTD 101 is an introduction to the Finite-Difference Time-Domain Method for Electromagnetics. We will walk you through the basics of setting up and running electromagnetic simulations using the FDTD method. Through this course, you will gain a knowledge of the fundamental concepts behind electromagnetic simulation as well as many advanced topics worth considering when you set up your simulations.
+
+`Lecture 1: Introduction to FDTD Simulation <https://www.flexcompute.com/fdtd101/Lecture-1-Introduction-to-FDTD-Simulation/>`_
+
+`Lecture 2: Using FDTD to Compute a Transmission Spectrum <https://www.flexcompute.com/fdtd101/Lecture-2-Using-FDTD-to-Compute-a-Transmission-Spectrum/>`_
+
+`Lecture 3: Applying FDTD to Photonic Crystal Slab Simulation <https://www.flexcompute.com/fdtd101/Lecture-3-Applying-FDTD-to-Photonic-Crystal-Slab-Simulation/>`_
+
+`Lecture 4: Prelude to Integrated Photonics Simulation: Mode Injection <https://www.flexcompute.com/fdtd101/Lecture-4-Prelude-to-Integrated-Photonics-Simulation-Mode-Injection/>`_
+
+`Lecture 5: Modeling dispersive material in FDTD <https://www.flexcompute.com/fdtd101/Lecture-5-Modeling-dispersive-material-in-FDTD/>`_
+
+`Lecture 6: Introduction to perfectly matched layer (PML) <https://www.flexcompute.com/fdtd101/Lecture-6-Introduction-to-perfectly-matched-layer/>`_
+
+`Lecture 7: Time step size and CFL condition in FDTD <https://www.flexcompute.com/fdtd101/Lecture-7-Time-step-size-and-CFL-condition-in-FDTD/>`_
+
+`Lecture 8: Numerical dispersion in FDTD <https://www.flexcompute.com/fdtd101/Lecture-8-Numerical-dispersion-in-FDTD/>`_
+
+`Lecture 9: Dielectric constant assignment on Yee grids <https://www.flexcompute.com/fdtd101/Lecture-9-Dielectric-constant-assignment-on-Yee-grids/>`_
+
+`Lecture 10: Introduction to subpixel averaging <https://www.flexcompute.com/fdtd101/Lecture-10-Introduction-to-subpixel-averaging/>`_
+
+`Lecture 11: Introduction to Inverse Design In Photonics <https://www.flexcompute.com/fdtd101/Lecture-11-Introduction-to-Inverse-Design-In-Photonics/>`_
```

Comment:
Perhaps leave these out of here as they are in inversedesign.rst

Reply:
Oh just realized they are the same lectures. Will change that.

Reply:
Actually since all lectures are on the FDTD101 page, should we just have one lecture here too without separating out the inverse design? Are all future lectures to be added to the same page?

Reply:
either way. they will be split eventually. we just didn't have time yet to make a whole new website for inv des

Reply:
I see. Then in that case I will have them separated here and then update the links once they have a separate website.